Blockzilla/SettingsViewController.swift

```diff
@@ -327,28 +333,30 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         let subtitle: String
         
         switch biometryType {
-            case .faceID:
-                label = UIConstants.strings.labelFaceIDLogin
-                subtitle = String(format: UIConstants.strings.labelFaceIDLoginDescription, AppInfo.productName)
-            case .touchID:
-                label = UIConstants.strings.labelTouchIDLogin
-                subtitle = String(format: UIConstants.strings.labelTouchIDLoginDescription, AppInfo.productName)
-            default:
-                // Unknown biometric type
-                return nil
+        case .faceID:
+            label = UIConstants.strings.labelFaceIDLogin
+            subtitle = String(format: UIConstants.strings.labelFaceIDLoginDescription, AppInfo.productName)
+        case .touchID:
+            label = UIConstants.strings.labelTouchIDLogin
+            subtitle = String(format: UIConstants.strings.labelTouchIDLoginDescription, AppInfo.productName)
+        default:
+            // Unknown biometric type
+            return nil
         }
         
         let toggle = BlockerToggle(label: label, setting: SettingsToggle.biometricLogin, subtitle: subtitle)
         toggle.toggle.isEnabled = biometryType.hasIdentities
         return toggle
     }
-
-    fileprivate func toggleForIndexPath(_ indexPath: IndexPath) -> BlockerToggle {
-        var index = (indexPath as NSIndexPath).row
-        for i in 0..<(indexPath as NSIndexPath).section {
-            index += tableView.numberOfRows(inSection: i)
-        }
-        guard let toggle = toggles[index] else { return BlockerToggle(label: "Error", setting: SettingsToggle.blockAds) }
+    
+    fileprivate func toggleForIndexPath(_ indexPath: IndexPath) -> BlockerToggle? {
+        // Necessary check for func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath)
+        let sec: Int = (indexPath as NSIndexPath).section
```

Comment:
Not necessary to cast to NSIndexPath. I think you can just do `let sec = indexPath.section`. It's already of type Int.